src/java.base/share/classes/java/util/Locale.java

```diff
@@ -1079,6 +1081,7 @@ private static Locale initDefault() {
     private static Locale initDefault(Locale.Category category) {
         Properties props = GetPropertyAction.privilegedGetProperties();
 
+        Locale defaultLocale = Locale.defaultLocale;
```

Comment:
I'd use a different local variable name so that it won't clash with the field name. The same holds for other locations (`isoLanguages` and `languageTag`)

Reply:
Done!